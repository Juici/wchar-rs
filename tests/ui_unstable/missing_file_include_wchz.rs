use wchar::{include_wchz, wchar_t};

const MISSING_FILE: &[wchar_t] = include_wchz!("missing_file");

fn main() {}
