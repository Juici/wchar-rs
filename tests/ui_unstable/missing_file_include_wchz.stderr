error: couldn't read $DIR/tests/ui_unstable/missing_file: No such file or directory (os error 2)
 --> $DIR/missing_file_include_wchz.rs:3:48
  |
3 | const MISSING_FILE: &[wchar_t] = include_wchz!("missing_file");
  |                                                ^^^^^^^^^^^^^^
